src/components/CreateResetPassword/components/PasswordRequirementsList/queries/usePasswordRules.js

```diff
@@ -0,0 +1,35 @@
+import ky from 'ky';
+import { useQuery } from 'react-query';
+
+import { useStripes } from '../../../../../StripesContext';
+
+const usePasswordRules = () => {
+  const { locale = 'en', tenant, url } = useStripes().okapi;
+
+  const kyInstance = ky.create({
+    prefixUrl: url,
+    hooks: {
+      beforeRequest: [
+        request => {
+          request.headers.set('Accept-Language', locale);
+          request.headers.set('X-Okapi-Tenant', tenant);
+        }
+      ]
+    },
+    retry: 0,
+    timeout: 30000,
+  });
+
+  const { data } = useQuery(
+    ['requirements-list'],
+    async () => {
+      return kyInstance.get('tenant/rules').json();
```

Comment:
Don't we need a `limit` clause on the query like `tenant/rules?limit=100` in case there are more than 10 rules? There are at least 11 in folio-snapshot and bugfest at present.